streams/src/test/java/org/apache/kafka/streams/processor/internals/ClientUtilsTest.java

```diff
@@ -108,75 +110,73 @@ public class ClientUtilsTest {
 
     @Test
     public void fetchCommittedOffsetsShouldRethrowKafkaExceptionAsStreamsException() {
-        final Consumer<byte[], byte[]> consumer = EasyMock.createMock(Consumer.class);
-        expect(consumer.committed(PARTITIONS)).andThrow(new KafkaException());
-        replay(consumer);
+        @SuppressWarnings("unchecked")
+        final Consumer<byte[], byte[]> consumer = mock(Consumer.class);
```

Comment:
nit: I think here it makes sense to move the `Consumer` mock to the object fields to avoid the suppress.